apps/src/code-studio/assets/SpriteUpload.js

```diff
@@ -4,15 +4,51 @@ export default class SpriteUpload extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      file: null
+      fileData: null,
+      filePreviewURL: '',
+      filename: '',
+      category: ''
     };
   }
 
-  handleSubmit(event) {}
+  handleSubmit = event => {
+    event.preventDefault();
+    let xhr = new XMLHttpRequest();
 
-  handleChange = event => {
+    const onError = function() {
+      console.log(new Error(`${xhr.status} ${xhr.statusText}`));
```

Comment:
this may be future work, but we should display this error in the UI

Reply:
Since this is an internal tool, do you think we could just print the error straight to the UI or should we make it friendlier?

Reply:
yeah, i think we can just display the error in the UI -- curriculum may need our help to investigate, but it would make it easier for them to understand what happened